docs/usage/getting-started/20-running.md

```diff
@@ -0,0 +1,207 @@
+# Running Renovate
+
+As a Renovate end user, there are two main categories of use:
+
+- You self-host Renovate, e.g. by running the pre-built Docker image, or
+- Someone else is hosting Renovate, and you install/configure it for the repositories you choose
+
+If someone else is hosting Renovate for you, or you are using the WhiteSource Renovate App on GitHub, then you can skip ahead to the Installing Renovate into Repositories section.
+
+## Self-Hosting Renovate
+
+Self-hosting Renovate means that you are the "administrator" of the bot, which entails:
+
+- You provide infrastructure for it to run on,
+- You provision its global config,
+- You ensure it's running regularly
+
+### Available distributions
+
+#### npm Package (CLI)
+
+Renovate's Open Source CLI is built and distributed as the npm package `renovate`.
+You can run this directly in any Node.js environment - even via `npx` - and it will process all the repositories it is configured with, before exiting.
+If you need to support any non-npm lock files like Bundler, Go Modules or Poetry then you'll need to make sure they are pre-installed in the same environment alongside Renovate.
```

Comment:
```suggestion
When you install Renovate from npm it naturally does not come bundled with any third party tools or languages such as Ruby, Python, Composer, Bundler, Poetry, etc.
Therefore if you need Renovate to support any non-npm lock files like Bundler then you'll need to make sure all required third party tools are pre-installed in the same environment alongside Renovate before you run it.
```

Reply:
Better! 😄 I cannot apply review suggestions, so you'll have to do that yourself! 😉